### PR TITLE
Corrections to Irminger deploy csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
2 updates to Irminger deploy csvs:

GI02HYPM_Deploy.csv - deployment 4; longitude was incorrectly noted on metadata review sheet and carried over to metadata review deploy_csv updates.  Error with longitude for deployment 4 was discovered by Russ D. and confirmed by RT and SW.  Corrected from -39.9256 to -39.5296

GI05MOAS-GL365_Deploy.csv - calculation error found during QLR review & corrected (for the first deployment of this glider on Irminger 7 cruise).  Corrected from -39.300067 to -39.300006

### CODE REVIEW

File: deployment/GI02HYPM_Deploy.csv [hunk 1]
```diff
@@ -26,15 +26,15 @@ AR7-01,,AR21,,GI02HYPM-WFP02-04-CTDPFL000,3,1,2016-07-11T14:29:00,2017-08-10T10:
...
+AR21,,AR30-03,,GI02HYPM-WFP02-01-FLORDL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-FLORDL-02350,59.9733,-39.5296,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-WFP02-03-DOSTAL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,59.9733,-39.5296,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-WFP02-04-CTDPFL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,59.9733,-39.5296,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-WFP02-05-VEL3DL000,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,59.9733,-39.5296,,2560,2659,
+AR21,,AR30-03,,GI02HYPM-MPM01-02-ZPLSGA009,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,,CGINS-ZPLSGA-55107,59.9733,-39.5296,,150,2659,upward-looking
+AR21,,AR30-03,,GI02HYPM-MPM01-02-ZPLSGA010,4,1,2017-08-06T13:45:00,2018-06-16T06:32:00,CGMGI-02HYPM-00004,,CGINS-ZPLSGA-55108,59.9733,-39.5296,,150,2659,downward-looking
```
Comment (johnmlund) [inline deployment/GI02HYPM_Deploy.csv:37]: looks good

File: deployment/GI05MOAS-GL365_Deploy.csv [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-AR46,,AR46,,GI05MOAS-GL365-02-DOSTAM000,3,1,2020-08-15T16:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-DOSTAM-00281,59.9267,-39.300067,,1000,1000,GPS issue
-AR46,,AR46,,GI05MOAS-GL365-04-CTDGVM000,3,1,2020-08-15T16:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-CTDGVM-09061,59.9267,-39.300067,,1000,1000,GPS issue
+AR46,,AR46,,GI05MOAS-GL365-00-ENG000000,3,1,2020-08-15T16:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGCON-GLDROO-00365,59.9267,-39.300006,,1000,1000,GPS issue
+AR46,,AR46,,GI05MOAS-GL365-01-FLORDM000,3,1,2020-08-15T16:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-FLORDM-03124,59.9267,-39.300006,,1000,1000,GPS issue
+AR46,,AR46,,GI05MOAS-GL365-02-DOSTAM000,3,1,2020-08-15T16:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-DOSTAM-00281,59.9267,-39.300006,,1000,1000,GPS issue
+AR46,,AR46,,GI05MOAS-GL365-04-CTDGVM000,3,1,2020-08-15T16:18:00,2020-08-16T00:49:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-CTDGVM-09061,59.9267,-39.300006,,1000,1000,GPS issue
```
Comment (johnmlund) [inline deployment/GI05MOAS-GL365_Deploy.csv:5]: change looks right

**Review by johnmlund — APPROVED** [review]

File: deployment/GI02HYPM_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/GI02HYPM_Deploy.csv:37]: Looks good

File: deployment/GI05MOAS-GL365_Deploy.csv (hunk 2 above)
Comment (cdobs) [inline deployment/GI05MOAS-GL365_Deploy.csv:5]: Looks good

**Review by cdobs — APPROVED** [review]
Looks good to me

**Pull request merged**